### ISSUE TITLE
Abitur Aktualisierung der Punktsumme im Block II funktioniert nicht

### ISSUE THREAD

**Issue by beaumart** [SVWS-NRW/Schild3-BetaTest#1046]
Wenn SuS mehrere Existenzprüfungen ablegt sollte nach jeder eingegebenen Prüfung die Punktsumme in Block II aktualisiert werden. In Schild2 funktioniert das so. Vgl. Screenshot.
![AbiturZwischenergebnisse](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/50887283/a8713f72-850a-453d-97cc-1cb6477d9d14)

**Comment by AnneSchueller** [comment]
Hier muss man erst nochmal Gesamtergebnis berechnen anklicken. Ein direktes Eintragen der Punkte wäre etwas komfortabler.

**Comment by beaumart** [comment]
Das kann ich leider nicht bestätigen, im oberen Bereich wird das Ergebnis aktualisiert (im Beispiel oben auf 79 Punkte) aber unten im Block II bleiben 60Punkte, auch nach Betätigung des Buttons „Gesamtergebnis berechnen“, stehen, siehe oben: rot eingekreist.

**Comment by AnneSchueller** [comment]
Ich hoffe wir reden nicht aneinander vorbei:
Wenn man die Punkte der mündlichen Prüfung in SchILD3 eingibt und anschließend erneut auf "Gesamtergebnis berechnen" klickt, werden die Punkte unten eingetragen:

![image](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/79116896/6f5dca79-d090-48ff-bb9d-9202b9592f70)

**Comment by beaumart** [comment]
Hallo Anne,
bei mir wird in Block II die Punktzahl nicht aktualisiert, wenn ich die Schaltfläche „Gesamtergebnis berechnen“ klicke. Vgl. mein Screenshot.

**Comment by AnneSchueller** [comment]
Ist das eine Gym-Datenbank?

**Comment by beaumart** [comment]
Nein, eine Gesamtschule-DBVon meinem iPad gesendetAm 25.06.2024 um 19:39 schrieb AnneSchueller ***@***.***>:﻿
Ist das eine Gym-Datenbank?

**Comment by AnneSchueller** [comment]
Ich habe das nun auch einmal mit eine GE-DB getestet und kann das nicht reproduzieren.
Ein Klick auf "Gesamtergebnis berechnen" ändern gleichzeitig die rot umrandeten Punkte 
![image](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/79116896/d1daa258-f44a-41fb-9d6d-43cdc9a84b3c)

**Comment by beaumart** [comment]
Hallo Anne,
der Unterschied zu meinem Notenbild ist, dass mein Schüler das Abitur noch nicht bestanden hat und erst zwei der drei angesetzten Prüfungen abgelegt hat.

**Comment by AnneSchueller** [comment]
Stimmt, dann kann ich das auch reproduzieren. Die Punkteänderung erfolgt dann nur wenn man erneut auf "Gesamtergebnis berechnen" klickt. Das war in Schild2 komfortabler.
